doc/docs/overview.md

```diff
@@ -3,3 +3,215 @@
+
+    ![Selection section](./screenshots/ppk2_selection_zoom.png "Selection section")
+
+    You can make a selection in the measurement chart by pressing **Shift** and dragging the left mouse button. The selected part is highlighted in grey and included between two grey sliders.
+
+    ![Example of selection](./screenshots/ppk2_measurement_chart.png "Example of selection")
+
+### Digital channels
+
+The digital signals are visible in the charting section below the current measurement.
+
+![Digital channels](./screenshots/ppk2_digital_channels.png "Digital channels")
+
+The digital signals are connected to the PPK2's Logic port as described in [Logic port](https://docs.nordicsemi.com/bundle/ug_ppk2/page/UG/ppk/logic_port.html). To view the digital values, enable digital channels and zoom in on the main chart until the values are visible.
+
+### Advanced controls
+
+The advanced control panel gives you access to filter tuning and lets you adjust gains for all individual ranges.
+
+Press **CTRL-ALT-SHIFT-A** to access the advanced control area of the device panel.
+
+#### Session data
+
+![Advanced controls for session data](./screenshots/ppk2_advanced_session_data.png "Advanced controls for session data")
+
+Here you can set the root directory where the sampling file will be saved by default and define how much HDD space you want to keep free when you start sampling.
+The sampling will automatically stop when the sampling temporary file would become so big as to have to take space from the defined amount.
+
+#### Gains
+
+![Advanced controls for gains](./screenshots/ppk2_advanced_gains.png "Advanced controls for gains")
+
+If any of the ranges (see [Measurement resolution](https://docs.nordicsemi.com/bundle/ug_ppk2/page/UG/ppk/ppk_measure_resolution.html#ppk_measure_resolution__table_b3l_3ty_bdb)) has an offset, use these controls to add a positive or negative gain to the calculated measurement values.
+
+#### Spike filter
+
+![Advanced controls for spike filter](./screenshots/ppk2_advanced_spike.png "Advanced controls for spike filter")
+
+Whenever a dynamic range switching occurs, induced inductance may cause the first samples to be higher than the actual value.
+
+Use the sliders to set the following:
+
+- Samples to smooth - The number of samples after a dynamic range switch to apply the filer.
+- Coefficient for range 1–4 - The magnitude of the spike filter for range 1–4. The higher the value, the more filtering will be applied.
+- Coefficient for range 5 - The magnitude of the spike filter for range 5. The higher the value, the more filtering will be applied.
+
+#### Voltage limit
+
+![Advanced controls for voltage limit](./screenshots/ppk2_advanced_voltage.png "Advanced controls for voltage limit")
+
+Use this control to set the voltage limit for the measurement.
+
+## Feedback tab
+
+The Feedback tab lets you send feedback about nRF Connect Power Profiler application to the application development team.
+
+## About tab
```

Comment:
Can we also add to the "In this app" shortcut list:

- Select measurement range: SHIFT + LMB and drag mouse
- Scroll measurement chart: ALT + mouse wheel

Also, some entries in the shortcuts are currently duplicated, @kylebonnici .